docs/release-notes.asciidoc

```diff
@@ -56,6 +57,7 @@ This section summarizes the changes in each release.
 :issue: https://github.com/elastic/kibana/issues/
 :pull: https://github.com/elastic/kibana/pull/
```

Comment:
```suggestion
```

Reply:
Same comment as above: no need to remove this attribute definition here, and it might even break things. https://github.com/elastic/security-docs/pull/4976 will handle all the attribute clean-up.